Calculator/Calculator/Model/FormManager.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FormManager.swift
+//  Calculator
+//
+//  Created by mint, Whales on 2023/06/14.
+//
+
+import Foundation
+
+enum FormManager {
+    static let numberFormatter = NumberFormatter()
+    
+    static var configuredNumberFormatter : NumberFormatter {
+        self.numberFormatter.numberStyle = .decimal
+        self.numberFormatter.maximumFractionDigits = 19
+        self.numberFormatter.maximumIntegerDigits = 20
+        
+        return self.numberFormatter
+    }
```

Comment:
연산 프로퍼티는 값에 접근하는 순간마다 연산을 해주기 때문에 이대로라면 사용할 때 마다 똑같은 값으로 계속 설정해주겠네요!
좋은 방법이지만, 매번 사용할 때 마다 불필요한 연산이 되는 문제가 있네요.
일반적으로 한번 설정한 후 계속 그 값을 사용하는 경우에는 연산 프로퍼티보다는 클로저를 사용해 초기화 해주면 좋습니다.
```suggestion
    static let configuredNumberFormatter: NumberFormatter = {
        let numberFormatter = NumberFormatter()
        numberFormatter.numberStyle = .decimal
        numberFormatter.maximumFractionDigits = 19
        numberFormatter.maximumIntegerDigits = 20
        
        return numberFormatter
    }()
```

Reply:
연산프로퍼티는 저장프로퍼티를 이용해야 한다는 글을 보고
따로 마련된 저장프로퍼티를 써야만 하는 줄 알았는데
이렇게 연산프로퍼티 로직 안에 쓸 수도 있군요!!!
좋은 방법 감사드립니다 원비!!!!🙇🏻‍♀️